packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.scss

```diff
@@ -22,6 +22,7 @@
 			}
 
 			.tc-badge-value-unit {
+				cursor: pointer;
```

Comment:
This should be a button. This is not a pointer because it's a span, so it's not accessible (no pointer, no focus, no activation via keyboard)
The right fix is to turn it into a button and fix the style, you'll have your pointer by default